src/lib/system-info.ts

```diff
@@ -23,13 +18,8 @@ export async function getStorageInfo(): Promise<{
 	// First we find the block device which the data partition is part of
 	for (const partition of fsInfo) {
 		if (partition.mount === '/data') {
-			const match = partition.fs.match(blockDeviceRegex);
```

Comment:
I think is the right decision to remove the match. This seems to restrict only to balena block devices, this was bound to bite us sooner or later.